lib/api/multipartDelete.js

```diff
@@ -117,6 +120,7 @@ function multipartDelete(authInfo, request, log, callback) {
                     pushMetric('abortMultipartUpload', log, {
                         authInfo,
                         bucket: bucketName,
+                        byteLength: partSizeSum,
```

Comment:
No. UtapiClient will use the value to decrement the `storageUtilized` metric. Similar to how DELETE operations are handled. 😄